libgnucash/app-utils/calculation/fin.c

```diff
@@ -149,9 +149,9 @@
  *
  * 1)
  *                                                                 FV*
- *          1   2   3   4   .   .   .   .   .   .   .   .   .   n �
- * Period ���������������������������������������������������������
- *        �
+ *          1   2   3   4   .   .   .   .   .   .   .   .   .   n �
+ * Period ���������������������������������������������������������
```

Comment:
Are you using a different encoding than utf-8, perhaps Microsofts ancestor of utf-16?

Reply:
i honestly don't know how to fix this encoding issue. I've googled around but can't seem to find the right search term. 

Anyway,  I've manually edited the changes in vim, saved, and then squashed; theoretically the changes should cancel each other out. But instead the diff shows I'm overwriting with my changes which are identical to the maint branch?? Weird. 

![image](https://user-images.githubusercontent.com/4140247/214811658-b555e5f1-f0e2-420c-8857-414d79d5f3ec.png)

Sorry for the noise. Maybe a merge dev can swoop in and cherrypick the appropriate changes (without the encoding nonsense my IDE introduced in this commit?)